src/ports/postgres/modules/stats/correlation.sql_in

```diff
@@ -207,8 +203,9 @@ Result:
 </pre>
 
 @par Notes
-Current implementation ignores a row that contains NULL entirely. This means
-any correlation in such a row (with NULLs) does not contribute to the final answer.
+
+WARNING: Rows with NULL values will not be ignored. Null values will be
+replaced by the mean of their respective columns.
```

Comment:
1. The `@par Notes` adds formatting. Don't need the warning tag. 
2. I suggest removing the "Rows with NULL values will not be ignored."

Reply:
Here is a blog post on the dangers of mean imputation
https://www.theanalysisfactor.com/mean-imputation/

Can we point to some literature (and put in the user docs) that back up our plan to use mean imputation?